checker/examples/Makefile

```diff
@@ -4,7 +4,6 @@ all: compile
 
 compile:
 	$(JAVAC) *.java
-	$(JAVAC) units-extension/*.java
```

Comment:
Why is this no longer needed? Doesn't this at least check that the code compiles?